MetaDataset.py

```diff
@@ -847,6 +847,7 @@ def __init__(self,
     # This will only initialize datasets needed for features occurring in data_map
     self.datasets = {key: init_dataset(datasets[key]) for key in self.dataset_keys}
 
+    self._can_use_num_seqs = None  # whether Dataset._num_seqs of all sub-datasets is available, set in init_seq_order()
```

Comment:
Why do you need the attrib? I thought this is enough as a local variable in `init_seq_oder`? (As I explained, it is always bad to have too much shared state. I don't see why this is needed here.)